addons/mail/models/mail_render_mixin.py

```diff
@@ -184,7 +185,7 @@ def _sub_relative2absolute(match):
         return html
 
     @api.model
-    def _render_encapsulate(self, layout_xmlid, html, add_context=None, context_record=None):
+    def _render_encapsulate(self, layout_xmlid, html=None, add_context=None, context_record=None):
```

Comment:
Why a None default value ? It makes no sense to encapsulate a None. What is the underlying issue ?